gradio_vc_app.py

```
@@ -256,16 +256,50 @@ def convert(src_wav_path, ref_wav_path, seed_num):
+            rtf = conversion_time / duration
+            
+            info_text = f"""✅ **Conversion complete!**
+- Audio duration: {duration:.2f} seconds
+- Conversion time: {conversion_time:.2f} seconds
+- Real-time factor (RTF): {rtf:.2f}x
+- Sample rate: {sr:,} Hz
+- Click the download button above to save"""
+            
+            return result, gr.update(value=info_text, visible=True)
+        else:
+            return None, gr.update(value="❌ Voice conversion failed", visible=True)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)